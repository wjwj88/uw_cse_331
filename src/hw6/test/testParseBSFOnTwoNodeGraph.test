# test parser and BSF two nodes graph  generic version-Jun Wang 5:00

# build graph
LoadGraph twoNodeGraph twoNode.tsv

# check graph is built correctly
ListNodes twoNodeGraph 
ListChildren twoNodeGraph Amanda
ListChildren twoNodeGraph Jack

# check BFS
FindPath twoNodeGraph Amanda Amanda
FindPath twoNodeGraph Amanda Jack
FindPath twoNodeGraph Jack Amanda 