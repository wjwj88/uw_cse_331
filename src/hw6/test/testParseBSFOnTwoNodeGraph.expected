# test parser and BSF two nodes graph  generic version-Jun Wang 5:00

# build graph
loaded graph twoNodeGraph

# check graph is built correctly
twoNodeGraph contains: Amanda Jack
the children of Amanda in twoNodeGraph are: Jack(Seattle)
the children of Jack in twoNodeGraph are: Amanda(Seattle)

# check BFS
path from Amanda to Amanda: 
path from Amanda to Jack: 
Amanda to Jack via Seattle
path from Jack to Amanda: 
Jack to Amanda via Seattle